File: src/main/Java/boj_step4/boj2562.java
import java.io.*;

public class boj2562 {
    public static void main(String[] args) throws IOException {
        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter writer = new BufferedWriter(new OutputStreamWriter(System.out));
        int size = 9;
        int[] list = new int[size];
        for (int i = 0; i < size; i++) list[i] = Integer.parseInt(reader.readLine());
        int max = Integer.MIN_VALUE;
        int index = 1;
        int count = 0;
        for (int it: list) {
            if (it > max) {
                max = it;
                count = index;
            }
            index++;
        }
        writer.append(String.valueOf(max));
        writer.newLine();
        writer.append(String.valueOf(count));
        writer.close();
    }
}

File: src/main/kotlin/others/boj2562.kt
import java.io.BufferedReader
import java.io.BufferedWriter
import java.io.InputStreamReader
import java.io.OutputStreamWriter

fun main() {
    val r = BufferedReader(InputStreamReader(System.`in`))
    val w = BufferedWriter(OutputStreamWriter(System.out))
    var arr = mutableListOf(0)
    var cnt = 9
    while (cnt-- > 0) arr.add(r.readLine()!!.toInt())
    w.write(arr.maxOrNull().toString() + "\n")
    w.write(arr.indexOf(arr.maxOrNull()).toString())
    w.close()
}